libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1291,6 +1291,16 @@ float AP_OSD_AbstractScreen::u_scale(enum unit_type unit, float value)
     return value * scale[units][unit] + (offsets[units]?offsets[units][unit]:0);
 }
 
+char AP_OSD_Screen::get_arrow_font_index(uint32_t angle)
```

Comment:
please make variable name angle_cd and make it int32_t so caller can supply either -18000 to 18000 or 0 to 36000, then move the wrap_360_cd() into get_arrow_font_index() and remove from callers

Reply:
done